SPECS/libnetfilter_log/libnetfilter_log.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
     "Signatures": {
-     "libnetfilter_log-1.0.1.tar.bz2": "74e0fe75753dba3ac114531b5e73240452c789a3f3adccf5c51217da1d933b21"
+     "libnetfilter_log-1.0.2.tar.bz2": "e3f408575614d849e4726b45e90c7ebb0e6744b04859555a9ce6ec40744ffeea"
     }
    }
```

Comment:
quick thing: can you add a newline to this file? I think that's the goal moving forward

Reply:
Added a newline